functions/Test-DbaDbCompression.ps1

```diff
@@ -58,22 +84,66 @@ function Test-DbaDbCompression {
-        with the recommendation of either Page or Row into and nicely formatted GridView
+
+        Returns results of all potential compression options for a single database DBName
+        with the recommendation of either Page or Row or NO_GAIN in a nicely formatted GridView
 
     .EXAMPLE
-        Test-DbaCompression -SqlInstance ServerA
+        $cred = Get-Credential sqladmin
+        Test-DbaCompression -SqlInstance ServerA -ExcludeDatabase MyDatabase -SqlCredential $cred
+
+        Returns results of all potential compression options for all databases except MyDatabase
+        on instance ServerA using SQL credentials to authentication to ServerA
+        Returns the recommendation of either Page, Row or NO_GAIN
+
+    .EXAMPLE
+        Test-DbaCompression -SqlInstance ServerA -Schema Test -Table MyTable
+
+        Returns results of all potential compression options for the Table Test.MyTable in instance ServerA
+        on ServerA and ServerB. Returns the recommendation of either Page, Row or NO_GAIN.
+        Returns a result for each partiton of any Heap, Clustered or NonClustered index
+
+    .EXAMPLE
+        Test-DbaCompression -SqlInstance ServerA, ServerB -MaxResultsPerDb 10
+
         Returns results of all potential compression options for all databases
-        with the recommendation of either Page or Row
+        on ServerA and ServerB. Returns the recommendation of either Page, Row or NO_GAIN.
+        Returns results for the top 10 partitions by TotalPages used per database.
+
+    .EXAMPLE
+        ServerA | Test-DbaCompression -Schema Test -MaxResultsPerDb 10 -RankBy UsedPages -FilterBy Table
+
+        Returns results of all potential compression options for all databases on ServerA containing a schema Test
+        Returns results for the top 10 Tables by Used Pages per database.
+        Results are split by Table, Index and Partition so more than 10 results may be returned.
+
+    .EXAMPLE
+        $servers = 'Server1','Server2'
+        $servers | Test-DbaCompression -Database DBName | Out-GridView
+
+        Returns results of all potential compression options for a single database DBName
+        on Server1 or Server2
+        Returns the recommendation of either Page, Row or NO_GAIN in a nicely formatted GridView
+
     .EXAMPLE
         $cred = Get-Credential sqladmin
-        Test-DbaCompression -SqlInstance ServerA -ExcludeDatabase Database -SqlCredential $cred
-        Returns results of all potential compression options for all databases
-        with the recommendation of either Page or Row
+
+        Test-DbaCompression -SqlInstance ServerA -Database MyDB -SqlCredential $cred -Schema Test -Table Test1, Test2
+
+        Returns results of all potential compression options for objects in Database MyDb on instance ServerA
+        using SQL credentials to authentication to ServerA. Returns the recommendation of
+        either Page, Row or NO_GAIN for tables with SchemA Test and name in Test1 or Test2
```

Comment:
Please remove line break, you can break it up by sentences if needed but sentence should be on single line.